src/test/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfigTest.java

```diff
@@ -84,6 +84,21 @@ public void testTopicDirFromTopicParts() {
     );
   }
 
+  @Test
+  public void testTopicDirCanContainNumber() {
```

Comment:
Add a testcase to test something that is smaller than the range of valid match groups - `${0}`, `${-1000}` or something.

Reply:
great edge case!